cmd/app.go

```diff
@@ -787,14 +789,18 @@ func createNamespace(c *cliclient.MasterClient, n string) error {
 			return err
 		}
 
+		nsID := ns.ID
 		startTime := time.Now()
 		for {
 			logrus.Debug(fmt.Sprintf("Namespace create wait - Name: %s, State: %s, Transitioning: %s", ns.Name, ns.State, ns.Transitioning))
 			if time.Since(startTime)/time.Second > 30 {
 				return fmt.Errorf("timed out waiting for new namespace %s", ns.Name)
 			}
-			ns, err = c.ClusterClient.Namespace.ByID(ns.ID)
-			if err != nil {
+			ns, err = c.ClusterClient.Namespace.ByID(nsID)
+			if e, ok := err.(*clientbase.APIError); ok && e.StatusCode == http.StatusForbidden {
```

Comment:
This check should be inside the `if err != nil {` if we don't have an error we don't need to type assert the error.
```
			ns, err = c.ClusterClient.Namespace.ByID(nsID)
			if err != nil {
				if e, ok := err.(*clientbase.APIError); ok && e.StatusCode == http.StatusForbidden {
					continue
				}
				return err
			}
```

Reply:
make sense